File: docs/components_page/components/button/flex_block.jl
using DashBootstrapComponents

button = html_div(
    [
        dbc_button("Button 1", className = "me-md-2"),
        dbc_button("Button 2", className = "me-md-2"),
        dbc_button("Button 3"),
    ],
    className = "d-grid gap-2 d-md-flex justify-content-md-end",
);

File: docs/components_page/components/button/flex_block.R
library(dashBootstrapComponents)

button <- div(
  list(
    dbcButton("Button 1", className = "me-md-2"),
    dbcButton("Button 2", className = "me-md-2"),
    dbcButton("Button 3")
  ),
  className = "d-grid gap-2 d-md-flex justify-content-md-end"
)
